src/Operation/Find.php

```diff
@@ -293,6 +294,38 @@ public function execute(Server $server)
         return $cursor;
     }
 
+    /**
+     * Explain the operation.
+     *
+     * @see Explainable::explain()
+     * @param Server $server
+     * @param $command
+     * @param array $options
+     * @return array|object
+     * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     */
+    public function explain($server, $command, $options = [])
+    {
+        if ( ! isset($options['verbosity'])) {
+            $options['verbosity'] = 'allPlansExecution';
+        }
+
+        $cmd = new Command(['explain' => ['find' => $this->collectionName, 'query' => $command], 'verbosity' => $options['verbosity']]);
```

Comment:
Is it correct to have `'query' => $command` here? I wasn't sure based on the [explain reference](https://docs.mongodb.com/manual/reference/command/explain/) since there are no examples for `find`.

Reply:
This didn't occur to me yesterday, but it's the same issue we have with explaining `update` and `delete` commands. Since those don't route through `executeCommand()`, PHPLIB never creates the actual command documents for those. We'll need logic to create command documents manually, which will be used purely for passing into the `explain` command.

For the time being, let's just focus on things that use `executeCommand()` (and related methods), such as Count.